docs/drafts/developer.md

```diff
@@ -173,23 +154,39 @@ v0.33.1, built 2023-06-28
 
 Typically, you'll want to press the space bar to have it open the UI in your web browser.
```

Comment:
This could probably be a bit more descriptive, like saying that at the end of the installation process the command output will prompt you to press spacebar to open the web UI which provides a useful overview of all the installed components.

Reply:
done in e379ecae